README.md

```diff
@@ -7,7 +7,7 @@
 Install the config and its dependencies:
 
 ```sh
-npm install --save-exact --save-dev eslint@^3.19.0 eslint-config-springload eslint-plugin-import@^2.7.0 eslint-plugin-jsx-a11y@^5.1.1
+yarn add --exact --save-dev eslint@^8.21.0 eslint-config-springload eslint-plugin-import@^2.26.0 eslint-plugin-jsx-a11y@^6.6.1
```

Comment:
I believe the `--save-dev` equivalent for yarn is (your choice of) `-D` or `--dev`

Reply:
It certainly is 👀 good spot, I thought I'd caught all of the npm -> yarn config conversions